bin/mocha-chrome

```diff
@@ -57,7 +57,7 @@ if (!/^(file|http(s?)):\/\//.test(file)) {
 }
 
 const mochaDefaults = {
-  reporter: 'spec',
+  reporter: flags.reporter || 'spec',
```

Comment:
shouldn't be a need for that, as it should be overwritten on line 63 in the `deepAssign` call. in fact, line 61 should be `useColors: true` for the same reason :D